packages/md2mld/md2mld.0.4.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Little cli tool to convert md files into mld files"
+maintainer: "Marcello Seri <marcello.seri@gmail.com>"
+authors: "Marcello Seri <marcello.seri@gmail.com>"
+license: "ISC"
+homepage: "https://github.com/mseri/md2mld"
+doc: "https://mseri.github.io/md2mld/"
+bug-reports: "https://github.com/mseri/md2mld/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "2.0"}
+  "base-bytes"
+  "omd" {= "2.0.0~alpha1"}
```

Comment:
This seems to be a rather strict constraint. Is that expected? Would it be not worth it to modify a little bit if needed and be compatible with the new alpha2 that has just been released?

Reply:
Yes, omd 2.0.0 is breaking the internal type representation. The first alpha broke compatibility with omd 1.x and the second alpha breaks compatibility with the first alpha. I will release a version compatible with alpha2 sometimes soon